tutoraspects/templates/aspects/apps/superset/pythonpath/openedx_sso_security_manager.py

```diff
@@ -31,6 +32,10 @@ def add_bearer_jwt_token(token, uri, headers, body, placement="header"):
     return uri, headers, body
 
 
+def create_clickhouse_username(lms_username):
+    return f"openedx-{lms_username}"
```

Comment:
For reference, this is the pattern Cairn uses. I opted not to change it, for backwards compat, but we still need to document the Cairn migration path and put in the required shims.